src/init.rs

```diff
@@ -0,0 +1,19 @@
+use anyhow::Result;
+use gettextrs::{setlocale, LocaleCategory};
+use log::info;
+use std::fs::File;
+use std::io::Write;
+
+/// Unset the locale for the current process.
+pub fn unset_locale() {
+    setlocale(LocaleCategory::LcAll, "");
+}
+
+/// Helper to adjust the OOM score of the currently running process.
+pub fn set_oom(score: &str) -> Result<()> {
+    // Attempt adjustment with best-effort.
+    if let Err(err) = File::create("/proc/self/oom_score_adj")?.write_all(score.as_bytes()) {
+        info!("Failed to adjust oom score {}", err)
```

Comment:
This function never returns an error, is this intentional?

Reply:
Yes, though in answering I've reconsidered. We need to allow permission denied to allow rootless conmon-rs, but we can error on the others